charts/truefoundry/templates/truefoundry-frontend-app/virtualservice.yaml

```diff
@@ -62,4 +62,15 @@ spec:
             host: {{ include "truefoundry-frontend-app.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
             port:
               number: {{ .Values.truefoundryFrontendApp.service.port }}
+    - match:
+        - uri:
+            prefix: "/flyte/"
+      rewrite:
+        uri: /
+      route:
+        - destination:
+            host: {{ tpl .Values.truefoundryFrontendApp.flyteServerHost . }}
+            port:
+              number: 8089
+          weight: 100
```

Comment:
1. this is WRONG. Please use the full path that I sent and NOT this. This will not work.
2. This change needs to be in helm chart. Infra charts are updated on release from helm-charts on its own. Please raise a PR separately.

Also, we need to be sure who populates the value of this
`.Values.truefoundryFrontendApp.flyteServerHost`

And why naming it this way? Please see the naming convention in the tfy-flyte-admin helm chart.

`.Values.truefoundryFrontendApp.tfyWorkflowAdminHost`